<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Game</class>
 <widget class="QWidget" name="Game">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1000</width>
    <height>800</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <widget class="QPushButton" name="retractButton">
   <property name="geometry">
    <rect>
     <x>820</x>
     <y>280</y>
     <width>111</width>
     <height>51</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>AaJHNDMCC (Non-Commercial Use)</family>
     <pointsize>24</pointsize>
    </font>
   </property>
   <property name="text">
    <string>悔棋</string>
   </property>
  </widget>
  <widget class="QPushButton" name="hintButton">
   <property name="geometry">
    <rect>
     <x>820</x>
     <y>360</y>
     <width>111</width>
     <height>51</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>AaJHNDMCC (Non-Commercial Use)</family>
     <pointsize>24</pointsize>
    </font>
   </property>
   <property name="text">
    <string>提示</string>
   </property>
  </widget>
  <widget class="QPushButton" name="exitButton">
   <property name="geometry">
    <rect>
     <x>820</x>
     <y>520</y>
     <width>111</width>
     <height>51</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>AaJHNDMCC (Non-Commercial Use)</family>
     <pointsize>24</pointsize>
    </font>
   </property>
   <property name="text">
    <string>退出</string>
   </property>
  </widget>
  <widget class="QPushButton" name="restartButton">
   <property name="geometry">
    <rect>
     <x>820</x>
     <y>440</y>
     <width>111</width>
     <height>51</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>AaJHNDMCC (Non-Commercial Use)</family>
     <pointsize>24</pointsize>
    </font>
   </property>
   <property name="text">
    <string>重新开始</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
